react/components/EXPERIMENTAL_TableTree/Tree/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { FC } from 'react'
+import React, { FC, useCallback } from 'react'
```

Comment:
Was this your update or something automatic?

Reply:
I did this, the function was being used but wasn't imported